src/main/java/rx/internal/operators/OnSubscribeOnAssembly.java

```diff
@@ -30,7 +31,7 @@
 
     final OnSubscribe<T> source;
     
-    final String stacktrace;
```

Comment:
Idk, it consumes memory and I've never used it during debugging. You really need it? I think you should be able to get ~normal output from `Arrays.toString(assemblyStacktrace)` in the debugger